cmd/kubernetesDeploy.go

```diff
@@ -148,6 +148,21 @@ func runHelmDeploy(config kubernetesDeployOptions, utils kubernetes.DeployUtils,
 		helmValues.add("imagePullSecrets[0].name", config.ContainerRegistrySecret)
 	}
 
+	utils.Stdout(stdout)
+
+	if config.DependencyUpdate {
+		dependencyUpdateParams := []string{
+			"dependency", "update",
+			config.ChartPath,
+		}
+
+		log.Entry().Info("Calling helm dependency update ...")
+		log.Entry().Debugf("Helm parameters %v", dependencyUpdateParams)
+		if err := utils.RunExecutable("helm", dependencyUpdateParams...); err != nil {
```

Comment:
Would also be possible to re-use [RunHelmDependency](https://github.com/SAP/jenkins-library/blob/master/pkg/kubernetes/helm.go#L378), but this is not done for other parts here in the code. We mimic the behaviour we find.